### PR TITLE
range loop refactoring

### PR DESCRIPTION
**Pull request by Joilnen**
and few reformat

### CODE REVIEW

File: OgreMain/src/OgreNumerics.cpp [hunk 1]
```diff
@@ -26,57 +26,59 @@ THE SOFTWARE.
 -----------------------------------------------------------------------------
 */
 
-#include "OgreStableHeaders.h"
```
Comment (paroj) [inline OgreMain/src/OgreNumerics.cpp:29]: please revert changes to this file. This does not improve code quality, but merely adds noise to the commit

Comment (Joilnen) [reply OgreMain/src/OgreNumerics.cpp]: did

File: OgreMain/src/OgreNumerics.cpp [hunk 2]
```diff
@@ -40,11 +40,11 @@ namespace Ogre
 
         int i, j;
 
-        for(j=0; j<n; j++)
+        for(j = 0; j < n; j++)
```
Comment (paroj) [inline OgreMain/src/OgreNumerics.cpp:43]: there are still unrelated changes in this file. please revert them as well.

**Review by paroj — CHANGES_REQUESTED** [review]
